src/content/wiki/buildings/cook.mdoc

```diff
@@ -10,12 +10,14 @@ The {% building /%} is where the Cook will cook food, provided they have the nec
 ## Interface
 
 {% building_gui_content_block_main order=1 /%}
-{% building_gui_content_block_crafting_recipes order=2 /%}
-{% building_gui_content_block_smelting_recipes order=3 /%}
-{% building_gui_content_block_stock order=4 /%}
-{% building_gui_content_block_fuel order=5 /%}
-{% building_gui_content_block_item_list order=6 header="food" imageKey="food" imageAlt="Food list" defaultOn=true %}
+{% building_gui_content_block_fuel order=2 /%}
+{% building_gui_content_block_item_list order=3 header="food" imageKey="food" imageAlt="Food list" defaultOn=true %}
 Listed here are food items that will be handed out by the {% worker name="cook" /%}. Disable any you don't want to be handed out or produced.
 However, if a colonist finds a disabled item, they'll still eat it - the {% worker name="cook" /%} just won't hand it out.
+
+Foods are sorted by what level of residence they are prefered. Aditionaly the background color indicates the quality of the food. 
+White is normal, bronze is better. Silver even better and Gold is the best food served.
```

Comment:
White is unrelated to Minecolonies food. Now it makes it look like white is one of the tiers, which is not true.

The bronze, silter, gold tiers are Minecolonies food tiers, these will be preferred by citizens and will have no additional happiness impact.

The white is any non Minecolonies food, these are not subject to the same checks and if given to citizens will cause an additional significant happiness impact.

So please separate out the white into an additional alinea or remove altogether.

Reply:
Thanks for the clarification, I could not figure out the effect of the tiers by a quick look at the code. 

I assume in the above explaination the happiness is switched around? Where non minecolonies food is just for nutrition and tiered food causes a positive happiness impact?

Reply:
No, Minecolonies food works just like regular food. However non Minecolonies food has a few drawbacks like worse happiness and less saturation gained from the food.
This is done to make vanilla food less useful and favoring Minecolonies specific food.